pkg/apis/submariner/v1alpha1/submariner_types.go

```diff
@@ -75,6 +75,8 @@ type SubmarinerStatus struct {
 	// Add custom validation using kubebuilder tags: https://book-v1.book.kubebuilder.io/beyond_basics/generating_crd.html
 }
 
+const ColorCodeBlue = "blue"
```

Comment:
I think what’s significant about this is that it’s our default colour code, not that it’s blue.

```suggestion
const DefaultColorCode = "blue"
```